Source/GameBaseFramework/Public/Gameplay/Components/GBFTriggerManagerComponent.h

```diff
@@ -67,7 +67,8 @@ class GAMEBASEFRAMEWORK_API UGBFTriggerManagerActivationPolicy_PercentageOfActor
     int GetExpectedActorsCount( const UObject * world_context, const TSubclassOf< AActor > & detected_actor_class ) const override;
 
 private:
-    UPROPERTY( BlueprintReadOnly, EditAnywhere, meta = ( AllowPrivateAccess ) )
+    // Defines a decimal percentage from 0 to 1
+    UPROPERTY( BlueprintReadOnly, EditAnywhere, meta = ( AllowPrivateAccess, UIMin = "0", UIMax = "1", ClampMin = "0", ClampMax = "1" ) )

```

Comment:
you can use `meta=(ForceUnits=%)`

Reply:
If I do that, it will show 0-1% (not 0-100%), which could make this confusing again